scriptmodules/ports/cgenius.sh

```diff
@@ -31,23 +31,16 @@ function build_cgenius() {
 
 function install_cgenius() {
     md_ret_files=(
-        'hqp'
-        'vfsroot/games'
-        'src/Build/LINUX/CGeniusExe'
+        'vfsroot'
+        'src/CGeniusExe'
     )
 }
 
 function configure_cgenius() {
-    addPort "$md_id" "cgenius" "Commander Genius" "pushd $md_inst; ./CGeniusExe; popd"
-
     mkRomDir "ports/$md_id"
-
+    mkUserDir "$home/.CommanderGenius"
+    
     moveConfigDir "$home/.CommanderGenius"  "$md_conf_root/$md_id"
 
-    mv "$md_inst/games" "$romdir/ports/$md_id/"
-    mv "$md_inst/hqp" "$romdir/ports/$md_id/"
-
-    ln -snf "$romdir/ports/$md_id/games" "$md_inst"
-
-    chown -R $user:$user "$romdir/ports/$md_id"
```

Comment:
@HerbFargus I will manually include your changes - but just to check - before we copied the games to the roms/ports dir and symlinked it - this is no longer required or ?

Reply:
The latest update includes a downloader, if someone is adding their own commander keen files they may have to add it to the config folder which I believe is where the downloaded game files end up. So it just depends on if the users are adding their own or using content from the cgenius downloader. I haven't really tested it with my own games with the latest changes

Reply:
The hqp files are built into the game files now though so those are definitely not necessary

Reply:
will test and see if anything needs to be done in regards to symlinking or config to make downloaded files go to the roms/ports etc

Reply:
ok thanks, sorry I haven't had more time to test. Just for reference the path to the game files is `~/.CommanderGenius/games` which per the moveconfig function symlinks to `/opt/retropie/configs/ports/cgenius/games` so if we want the user to be able to use their own game files we can symlink that folder as well to the `~/RetroPie/roms/ports/cgenius`  for them to be able to add them otherwise we can let them add them to the configs folder. I'm fine either way, I'll leave that up to you